src/trading/NewOrdersFetcher.sol

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (C) 2015 Forecast Foundation OU, full GPL notice in LICENSE
+ */
+
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+import 'ROOT/libraries/arrays/Bytes20Arrays.sol';
+import 'ROOT/libraries/Trading.sol';
+import 'ROOT/reporting/Interfaces.sol';
+import 'ROOT/trading/NewOrders.sol';
+
+
+/**
+ * @title OrdersFetcher
+ */
+// FIXME: This was named NewOrdersFetcher to prevent naming conflicts with ordersFetcher.se. Should be renamed to OrdersFetcher.
+contract NewOrdersFetcher is Controlled {
+    using Bytes20Arrays for bytes20[];
+
+    event Log(uint256 fxpPrice);
```

Comment:
I'm guessing this was added for testing?  Recommend removing if so, if not it is currently unused and should either be removed or used.

Reply:
Ah, good catch.  I'll remove that.